dotCMS/src/main/java/com/dotcms/rest/api/v1/page/PageResource.java

```diff
@@ -403,9 +412,16 @@ private Response getPageRender(final HttpServletRequest originalRequest,
     @POST
     @Produces({MediaType.APPLICATION_JSON, "application/javascript"})
     @Path("/{pageId}/layout")
+    @Operation(operationId = "postPageLayout",
+                summary = "Links template and HTML",
+                description = "Takes a saved template and links it to an HTML page.\n\n" +
+                                "Any pages with a template already linked will update with the new link.\n\n" +
+                                "Otherwise a new template will be created without making any changes to previous templates.\n\n" +
+                                "Returns the updated page view for specified page.\n\n",
+                tags = {"Page"})
     public Response saveLayout(@Context final HttpServletRequest request,
```

Comment:
I think you can replace the Response for a ResponseEntityView<Template> and also remove the exception handling; upper layer will take care about it

Reply:
Would this just involve removing the try/catch statements?